src/atom.cpp

```diff
@@ -2357,6 +2357,21 @@ void Atom::setup_sort_bins()
     return;
   }
 
+#ifdef LMP_GPU
+  if (userbinsize == 0.0) {   
```

Comment:
This is using obsolescent internal APIs. The modern way to do this is:
```
  if (userbinsize == 0.0) {
    auto ifix = dynamic_cast<FixGPU *>(modify->get_fix_by_id("package_gpu"));
    if (ifix) {
      const double subx = domain->subhi[0] - domain->sublo[0];
      const double suby = domain->subhi[1] - domain->sublo[1];
      const double subz = domain->subhi[2] - domain->sublo[2];
      binsize = ifix->binsize(subx, suby, subz, atom->nlocal, 0.5 * neighbor->cutneighmax);
    }
  }
```

Reply:
Made these changes.